cypress/e2e/CMS/$SiteSections/index.js

```diff
@@ -90,6 +90,21 @@ And('user selects {string} link under {string}', (children, parentName) => {
     cy.get(`a.menu-item__link:contains("${parentName}")`).parent().parent().find(`a:contains("${children}")`).click();
 });
 
-And('user clicks on {string} from  dropdown button under {string}', (deleteBtn, termName) => {
-    cy.get(`li.delete.dropbutton-action a:contains("${deleteBtn}")`).first().click({ force: true });
-});
\ No newline at end of file
+// And('user clicks on {string} from  dropdown button under {string}', (deleteBtn, termName) => {
+//     cy.get(`li.delete.dropbutton-action a:contains("${deleteBtn}")`).first().click({ force: true });
+// });
+
+And('user selects {string} operation for {string}', (editBtn, termName) => {
+    cy.get(`ul.dropbutton a:contains("${editBtn}")`).first().click({ force: true });
+});
+
+And('user checks {string} checkbox to set as a nav root', (setNavRoot) => {
+    cy.get('#edit-field-section-nav-root-wrapper > .js-form-item').find('input').check();
+});
+
+Then('left navigation does not display {string}', (leftNavItem) => {
+    cy.get(`nav[aria-label='Secondary navigation']`).find('a').contains(new RegExp(`^${leftNavItem}$`)).should('not.exist');
```

Comment:
do you need regex here? Doesn't it work with just a string?

Reply:
fixed